Readme.md

```
@@ -50,7 +50,7 @@ Choose one of the listed microcontrollers, but keep in mind that the ESP32 is re
 
   - ESP32 (recommended) - [AliExpress](https://s.click.aliexpress.com/e/_DdAe4Fl) or [Banggood][esp32-bg-shop] or [AliExpress][esp32-ali-shop]
   - ESP8266 - [AliExpress](https://s.click.aliexpress.com/e/_DCd13n1) or [Banggood][esp8266-bg-shop] or [AliExpress][esp8266-ali-shop]
-- Dupont Jumpers (female to male) - [AliExpress](https://s.click.aliexpress.com/e/_DmbrJsj) or [Banggood][dupont-jumpers-shop]
+- Dupont Jumpers (female to female) - [AliExpress](https://s.click.aliexpress.com/e/_DmbrJsj) or [Banggood][dupont-jumpers-shop]
```

Markdownlint

Expected: 2; Actual: 0
Inconsistent indentation for list items at the same level

(MD005, list-indent)